tests/src/test/scala/system/basic/WskRestBasicTests.scala

```diff
@@ -120,7 +120,11 @@ class WskRestBasicTests extends TestHelpers with WskTestHelpers {
     }
 
     wsk.action.create(packageName + "/" + actionName, defaultAction, annotations = actionAnnots)
-    val result = wsk.pkg.get(packageName)
+    val result = cacheRetry({
+      val p = wsk.pkg.get(packageName)
+      p.getFieldListJsObject("actions") should have size 1
```

Comment:
Retry only needs to be around pkg.get?

Reply:
kind of.. the pkg.get having 1 action.  in some cross-controller cache race conditions, the package is obtained but does not yet have the freshly created action.

Reply:
To clarify: I don't think there's an issue due to cross-controller cache race conditions here, but rather the view computing the summary of the package is eventually consistent. In either case, the retry is needed.